src/citrine/informatics/workflows/design_workflow.py

```diff
@@ -1,12 +1,10 @@
 from typing import Optional
 from uuid import UUID
 
-from deprecation import deprecated
 from citrine._rest.resource import Resource
 from citrine._serialization import properties
 from citrine._session import Session
 from citrine.informatics.workflows.workflow import Workflow
-from citrine.resources.workflow_executions import WorkflowExecutionCollection
 from citrine.resources.design_execution import DesignExecutionCollection
 from citrine._rest.ai_resource_metadata import AIResourceMetadata
 
```

Comment:
(having trouble commenting on the actually relevant line)
we can also remove `name = properties.String('display_name')` as well as the logic in the `pre_build` and `post_dump` that handles creating the `config` key

Reply:
good catch, thanks